code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -17,8 +17,6 @@
 	name = "Leporazine"
 	description = "Leporazine will effectively regulate a patient's body temperature, ensuring it never leaves safe levels."
 	color = "#C8A5DC" // rgb: 200, 165, 220
-	overdose_threshold = 30
-	metabolization_rate = 0.25 * REAGENTS_METABOLISM
```

Comment:
Instead of completly removing this, it would be better to just "buff" it, let's say:
overdose = 50  and  metabolize rate = 0.5